src/main/java/uk/gov/digital/ho/hocs/casework/domain/repository/JsonConfigFolderReader.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.digital.ho.hocs.casework.domain.repository;
```

Comment:
It might be worth looking at whether we want to move this into a package that we ingest within the repos.